test/common/router/config_impl_test.cc

```diff
@@ -36,18 +36,21 @@
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
-using testing::_;
-using testing::ContainerEq;
-using testing::Eq;
-using testing::MockFunction;
-using testing::NiceMock;
-using testing::Return;
-using testing::ReturnRef;
-
```

Comment:
Not sure about the movement here. I don't disagree, but should we set the same style for all?

Reply:
This change follows https://abseil.io/tips/119. My experience has been that folks prefer making style cleanups when modifying nearby code than cleanup-only PRs, which is why I haven't applied this everywhere.